aptos-move/framework/src/extended_checks.rs

```diff
@@ -30,6 +36,30 @@ const RESOURCE_GROUP_NAME: &str = "group";
 const RESOURCE_GROUP_SCOPE: &str = "scope";
 const VIEW_FUN_ATTRIBUTE: &str = "view";
 
+// top-level attribute names, only.
+pub fn get_all_attribute_names_set() -> &'static BTreeSet<String> {
+    const ALL_ATTRIBUTE_NAMES: [&str; 4] = [
```

Comment:
nit: you could use crate [maplit](https://docs.rs/maplit/latest/maplit/), which would allow you to do do
```
static KNOWN_ATTRIBUTES_SET: Lazy<BTreeSet<String>> = Lazy::new(|| {
    btreeset!{LEGACY_ENTRY_FUN_ATTRIBUTE, RESOURCE_GROUP, ...}
});
```

Or you just do:

```
static KNOWN_ATTRIBUTES_SET: Lazy<BTreeSet<String>> = Lazy::new(|| {
    ALL_ATTRIBUTE_NAMES.into_iter().collect()
});
```

Reply:
I missed this earlier.  I'll try it if/when I make more changes.

Reply:
I still think you should use at least my 2nd suggestion to make the initialization of KNOWN_ATTRIBUTES_SET more idiomatic and shorter.